app/services/wallet_transactions/validate_service.rb

```diff
@@ -31,14 +31,12 @@ def valid?
     attr_accessor :result, :args
 
     def valid_wallet?
-      current_wallet = Wallet.find_by(
-        id: args[:wallet_id],
-        customer_id: args[:customer_id],
-      )
+      organization = Organization.find_by(id: args[:organization_id])
 
-      return 'wallet_not_found' unless current_wallet
+      result.current_wallet = organization.wallets.find_by(id: args[:wallet_id])
```

Comment:
What about avoiding raising an exception if the organization is not found?

Reply:
I was thinking about that but organisation is probably the only record that is always present if user is logged in. So not really sure if these cases also needs to be covered? WDYT?

Reply:
Makes sense 👍